package.json

```diff
@@ -14,7 +14,11 @@
     "hallmark": "hallmark --fix",
     "dependency-check": "dependency-check . test/*.js bench/*.js",
     "prepublishOnly": "npm run dependency-check",
-    "prebuildify-cross-armv7": "prebuildify-cross --platform=linux --arch=arm --arm-version=7 -- -t 8.14.0 --napi --strip"
+    "prebuild-arm": "npm run prebuild-linux-arm && npm run prebuild-linux-arm64 && npm run prebuild-android-arm && npm run prebuild-android-arm64",
```

Comment:
Should we shorten this with e.g. [`npm-run-all`](https://github.com/mysticatea/npm-run-all)?

Reply:
Yes, good idea. (TIL about it)